docs/lint_errors.md

```diff
@@ -2,15 +2,19 @@
 
 This page contains detailed descriptions of the tests done by the [nf-core/tools](https://github.com/nf-core/tools) package. Linting errors should show URLs next to any failures that link to the relevant heading below.
 
-## Error #1 - File not found ## {#1}
-nf-core pipelines should adhere to a common file structure for consistency. The lint test looks for the following required files:
+## Error #1 - File not found / must be removed ## {#1}
+
+nf-core pipelines should adhere to a common file structure for consistency.
+
+The lint test looks for the following required files:
 
 * `nextflow.config`
   * The main nextflow config file
 * `Dockerfile`
   * A docker build script to generate a docker image with the required software
-* `.travis.yml` or `circle.yml`
-  * A config file for automated continuous testing with either [Travis CI](https://travis-ci.org/) or [Circle CI](https://circleci.com/)
+* Continuous integration tests with either [GitHub actions](https://github.com/features/actions), [Travis CI](https://travis-ci.com/) or [Circle CI](https://circleci.com/)
```

Comment:
I think we should rewrite this section, it's not that clear I think

Reply:
See https://github.com/nf-core/tools/pull/483/commits/ad243f3f44369e0546962fdf15172b6bb317e672

We cant point to a particular file until we completely deprecate `.travis.yml`. Maybe deal with that then?

Ive removed the references to Circle CI.